### DISCUSSION TITLE
PHP81 was removed, please use PHP82 or PHP83

### DISCUSSION THREAD

**Discussion by kodxana** [ZoeyVid/NPMplus#856] [Q&A] [answered]
After updating docker image today whole setup is broken and want start only error message is:
"PHP81 was removed, please use PHP82 or PHP83:

**Comment by Zoey2936** [comment]
yes alpine removed php8.1, so you need to switch to php8.2 or 8.3

**Reply by kodxana** [reply]
Do I need to rebuild docker container on my own or need to switch it so far using zoeyvid/npmplus:latest

**Reply by Zoey2936** [reply]
Can you send me your compose.yaml?

**Reply by kodxana** [reply]
or just editing env variable to true would work?
![image](https://github.com/ZoeyVid/NPMplus/assets/16674412/3b6dfd0e-0bf8-4049-82f4-6c84d62b866e)

**Reply by Zoey2936** [reply]
How do you update the container? The env PHP81 needs to be removed

**Reply by kodxana** [reply, marked as answer]
ok removed that venv variable and it works.
